### ISSUE TITLE
Database.get not running callback

### ISSUE THREAD

**Issue by THRYLLR** [TryGhost/node-sqlite3#1390]
I am using Version 5.0.0, and when I call `Database.get`, the callback parameter does nothing.

```javascript
module.exports = function(database) {
  //...
  database.get("[Insert Query Here]"/*The query is correct, I tested it in the SQLite3 CLI*/, [], (err, row) => {
    console.log((row || err) || "There is no row to log"); //This does not log anything
  });
};
```

**Comment by daniellockyer** [comment]
Would you be able to try with v5.0.3? Or provide a full reproduction case I can use? 🙂

**Comment by THRYLLR** [comment]
Thank you so much for getting back to me.

I unfortunately am unable to provide a full reproduction case; because in the 1 year and 182 days since I opened this issue, I gave up on this project, completely forgot about it, stopped using this library, closed my account on the service on which I was hosting my project, graduated schools, and learned four other programming languages.

Jokes aside, I'd love to fix this issue if only I could remember what it was in the first place. Thanks so much for helping!
